modules/postgres/src/test/scala/doobie/postgres/TypesSuite.scala

```diff
@@ -55,13 +54,16 @@ class TypesSuite extends munit.ScalaCheckSuite {
 
   def testInOut[A](col: String, a: A)(implicit m: Get[A], p: Put[A]) = {
     test(s"Mapping for $col as ${m.typeStack} - write+read $col as ${m.typeStack}") {
-      assertEquals(inOut(col, a).transact(xa).attempt.unsafeRunSync(), Right(a))
+      inOut(col, a).transact(xa).attempt
+        .assertEquals(Right(a))
```

Comment:
Looks like the reason why `.attempt` was used here in the current version in the first place is that it was a literal translation from Specs2 tests.

Feel free to keep it this way here too, but just in case, I'd suppose that without `attempt` it would work here and below just as fine:
```suggestion
      inOut(col, a).transact(xa)
        .assertEquals(a)
```